modules/concertBidAdapter.js

```diff
@@ -203,9 +204,8 @@ function hasOptedOutOfPersonalization() {
  * @param {BidderRequest} bidderRequest Object which contains any data consent signals
  */
 function consentAllowsPpid(bidderRequest) {
-  /* NOTE: We cannot easily test GDPR consent, without the
+  /* NOTE: We can't easily test GDPR consent, without the
    * `consent-string` npm module; so will have to rely on that
    * happening on the bid-server. */
-  return !(bidderRequest.uspConsent === 'string' &&
-           bidderRequest.uspConsent.toUpperCase().substring(0, 2) === '1YY')
+  return !(bidderRequest.uspConsent === 'string' && bidderRequest.uspConsent.toUpperCase().substring(0, 2) === '1YY');
```

Comment:
why are you checking for the second character https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/f8d659f58665b4e797b99b59079fd787a386eaaa/CCPA/US%20Privacy%20String.md?plain=1#L123

Reply:
This is checking the first 3 characters of the string; we do _not_ mark as consenting to PPID usage unless: (1) USPrivacy string _version_ matches what we expect, and (2) both `Notice to Opt-out` and `Opt-out Sale` are marked as consenting. (And we don't care about LSPA.)

Reply:
If the string is '1NY', the user is also opted out

Reply:
If I understand what you're saying, we should consider the user "opted-out" if the CCPA string starts `1NY`? If so, I suspect we didn't realise that was a possible value (i.e. can they have opted-out if they weren't notified of the opt-out option?). I guess we assumed that a `Y` in the third character implied a `Y` in the second character. If that's not the case, I'll happily update the PR to ignore the second character then. (And also, your original comment makes more sense to me now; thank you for that.)

Reply:
some people have started pushing '1NY' if the GPC headers in the browser indicate opt out.